tools/spiffsimg/README.md

```diff
@@ -0,0 +1,66 @@
+# spiffsimg - Manipulate SPI Flash File System disk images
+
+Ever wished you could prepare a SPIFFS image offline and flash the whole
```

Comment:
I'd much rather link to nodemcu.readthedocs.io/en/dev/en/spiffs/ (or ../../docs/en/spiffs.md) than repeating that content here.